examples/basic_voice.py

```diff
@@ -16,7 +16,7 @@
 from disnake.ext import commands
 
 # Suppress noise about console usage from errors
-youtube_dl.utils.bug_reports_message = lambda: ""
+youtube_dl.utils.bug_reports_message = lambda **_: ""
```

Comment:
```suggestion
youtube_dl.utils.bug_reports_message = lambda *args, **kwargs: ""
```
yt-dlp seems to use both `bug_reports_message(";")` and `bug_reports_message(before=";")`, so this should handle `*args` as well.